platform/linuxbsd/display_server_x11.cpp

```diff
@@ -57,9 +57,8 @@
 #define WM_NormalState 1L // window normal state
 #define WM_IconicState 3L // window minimized
 // EWMH
-#define _NET_WM_STATE_REMOVE 0L // remove/unset property
-#define _NET_WM_STATE_ADD 1L // add/set property
-#define _NET_WM_STATE_TOGGLE 2L // toggle property
+#define EWMH_NET_WM_STATE_REMOVE 0L // remove/unset property
+#define EWMH_NET_WM_STATE_ADD 1L // add/set property
```

Comment:
These actually come from the freedesktop specification: https://specifications.freedesktop.org/wm-spec/wm-spec-1.3.html

Reply:
I ended up reverting that rename, but I kep the removal of the unused _NET_WM_STATE_TOGGLE.